applications/nrf_desktop/doc/hid_provider_system_ctrl.rst

```diff
@@ -25,11 +29,30 @@ Module events
 Configuration
 *************
 
-You can enable the module using the :ref:`CONFIG_DESKTOP_HID_REPORT_PROVIDER_SYSTEM_CTRL <config_desktop_app_options>` Kconfig option.
-This option is enabled by default if the device supports HID system control reports.
-You can substitute the module with a custom HID system control report provider implementation.
+You can enable the default implementation of the HID provider using the :ref:`CONFIG_DESKTOP_HID_REPORT_PROVIDER_SYSTEM_CTRL <config_desktop_app_options>` Kconfig option.
+This option is enabled by default if the device uses HID provider events (:ref:`CONFIG_DESKTOP_HID_REPORT_PROVIDER_EVENT <config_desktop_app_options>`) and supports HID system control reports (:ref:`CONFIG_DESKTOP_HID_REPORT_SYSTEM_CTRL_SUPPORT <config_desktop_app_options>`).
+
+Alternatively, you can substitute the module with a custom HID system control report provider implementation.
 Enable the :ref:`CONFIG_DESKTOP_HID_REPORT_PROVIDER_SYSTEM_CTRL_ALT <config_desktop_app_options>` Kconfig option to use a custom HID system control report provider.
+The option disables the default HID system control report provider.
 Make sure to introduce the custom HID system control report provider if you enable this option.
+See the :ref:`nrf_desktop_hid_state_providing_hid_input_reports` documentation section of the HID state module for details.
+
+Default implementation
```

Comment:
We could use .. include:: for this section as the same is used in the hid_provider_consumer_ctrl.rst file - Default implementation.